OntologyMapper/src/Mappings/Mapped/Json/v0/Willow/mapped_json_v0_dtdlv2_willow.json

```diff
@@ -3323,7 +3323,7 @@
     },
     {
       "InputDtmi": "dtmi:mapped:core:OccupancySensor;1",
-      "OutputDtmi": "dtmi:com:willowinc:OccupancySensorEquipment;1"
+      "OutputDtmi": "dtmi:com:willowinc:OccupancySensor;1"
```

Comment:
Is this another change or a holdover from the last PR?

Reply:
Holdover